mozdownload/factory.py

```diff
@@ -55,14 +56,14 @@ def __init__(self, scraper_type, **kwargs):
         Tinderbox:
```

Comment:
Yes! Good catch. There are also params missing for Tinderbox specific things. I will have a look.